lib/routes/dewu/declaration.ts

```diff
@@ -0,0 +1,86 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { parseDate } from '@/utils/parse-date';
+import timezone from '@/utils/timezone';
+
+const typeMap = {
+    '1010580020': '技术变更',
+    '1014821004': '服务市场规则中心',
+    '1011202692': '规则变更',
+    '1010568195': '维护公告',
+};
+
+/**
+ *
+ * @param ctx {import('koa').Context}
+ */
+export const route: Route = {
+    path: '/declaration/:categoryId?',
+    categories: ['programming'],
+    example: '/dewu/declaration/1010580020',
+    parameters: { categoryId: '公告分类, 可在页面URL获取 默认为1010580020' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: '平台公告',
+    maintainers: ['zhijunchai'],
```

Comment:
```suggestion
    maintainers: ['blade0910'],
```
This one should be your GitHub ID.